updateStableIds/src/main/java/org/reactome/release/updateStableIds/UpdateStableIds.java

```diff
@@ -0,0 +1,171 @@
+package org.reactome.release.updateStableIds;
+
+import static org.gk.model.ReactomeJavaConstants.*;
+
+import java.util.*;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+public class UpdateStableIds {
```

Comment:
The class name should probably be changed to `StableIDUpdater` and the method `stableIdUpdater` should probably be changed to `updateStableIDs` to conform to more common conventions.

Reply:
Right, I forgot you pointed that out

Reply:
Remember to use `git mv` for file renaming - you'll probably need to do that first, if you want to preserve file history in git.